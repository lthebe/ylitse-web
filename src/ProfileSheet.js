import React from 'react';
import Typography from 'material-ui/Typography';

import Page from './Page';
import ProfileForm from './ProfileForm';

const ProfileSheet = () => (
    <Page>
        <Typography variant="headline" component="h3">
            Create account
        </Typography>
        <ProfileForm />
    </Page>
);

export default ProfileSheet;
