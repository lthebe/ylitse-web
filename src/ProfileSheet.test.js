import React from 'react';
import { shallow } from 'enzyme';
import ProfileSheet from './ProfileSheet';

describe('ProfileSheet', () => {
    test('renders correctly', () => {
        // ProfileSheet is a HOC, so diving
        const shallowWrapper = shallow(<ProfileSheet />).dive();
        expect(shallowWrapper).toMatchSnapshot();
    });
});
