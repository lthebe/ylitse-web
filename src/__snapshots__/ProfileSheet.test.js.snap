// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileSheet renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(Page)>
    <WithStyles(Typography)
      component="h3"
      variant="headline"
    >
      Create account
    </WithStyles(Typography)>
    <WithStyles(AccountForm) />
  </WithStyles(Page)>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <WithStyles(Typography)
          component="h3"
          variant="headline"
        >
          Create account
        </WithStyles(Typography)>,
        <WithStyles(AccountForm) />,
      ],
      "classes": Object {
        "root": "Page-root-1",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Create account",
          "component": "h3",
          "variant": "headline",
        },
        "ref": null,
        "rendered": "Create account",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <WithStyles(Typography)
            component="h3"
            variant="headline"
          >
            Create account
          </WithStyles(Typography)>,
          <WithStyles(AccountForm) />,
        ],
        "classes": Object {
          "root": "Page-root-1",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Create account",
            "component": "h3",
            "variant": "headline",
          },
          "ref": null,
          "rendered": "Create account",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
